### PR TITLE
testing/mame: new aport

### PR DESCRIPTION
**Pull request by tanertas**
http://mamedev.org
Multi Arcade Machine Emulator with GroovyMAME/Switchres/No-nag patchset.

### CODE REVIEW

**Comment by tanertas** [comment]
@ncopa had an early review on this PR with email [1]. I made suggested corrections with a version upgrade. I created a new PR from scratch because previous one (#4088) was on my master branch.

[1] http://lists.alpinelinux.org/alpine-aports/5362.html

File: testing/mame/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+license="GPL-2.0-or-later"
+depends="$pkgname-common"
+makedepends="coreutils dos2unix nasm clang-dev libxrandr-dev libxinerama-dev
+	alsa-lib-dev python2-dev py3-sphinx sdl2-dev sdl2_ttf-dev expat-dev zlib-dev
+	mesa-dev libxinerama-dev fontconfig-dev libjpeg-turbo-dev flac-dev lua5.3-dev
+	sqlite-dev portaudio-dev rapidjson-dev glm-dev portmidi-dev asio-dev"
+subpackages="$pkgname-data::noarch $pkgname-doc $pkgname-common::noarch $pkgname-lang
+	$pkgname-tools $pkgname-plugins::noarch $pkgname-arcade $pkgname-mess"
+_groovymame_patch=groovymame-${pkgver}.diff
+source="https://github.com/mamedev/${pkgname}/archive/${pkgname}${pkgver/.}.tar.gz
```
Comment (andypost) [inline testing/mame/APKBUILD]: please change source to `https://github.com/mamedev/mame/archive....` to easily get url of releases

File: testing/mame/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,165 @@
+# Contributor: Taner Tas <taner76@gmail.com>
+# Maintainer: Taner Tas <taner76@gmail.com>
+pkgname=mame
+pkgver=0.200
+pkgrel=0
+pkgdesc="Multi Arcade Machine Emulator with GroovyMAME/Switchres/No-nag patchset."
+url="http://mamedev.org"
```
Comment (andypost) [inline testing/mame/APKBUILD]: there's https version

File: testing/mame/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+source="https://github.com/mamedev/${pkgname}/archive/${pkgname}${pkgver/.}.tar.gz
+	$_groovymame_patch::https://drive.google.com/uc?export=download&id=1ruDLYXbD8Y1lSUZ4ZNZlzbWWqw0-Zi1O
+	mame.ini
+	midi.conf
+	fix-musl.patch
+	m68000_archopts.patch
+	nonag.patch
+	SubtargetArcade_0200u0.diff
+	"
+options="!check"
```
Comment (andypost) [inline testing/mame/APKBUILD]: please add comment why there's no tests

Comment (tanertas) [reply testing/mame/APKBUILD]: I made some cosmetic changes along with proposed ones from you.

**Comment by andypost** [comment]
I takes few hours to build... but finally somehow build went fine

So I think it good to go

**Comment by tanertas** [comment]
Thank you @andypost . I don't see armhf and s390x packages. Did they fail? Can I see build logs?

**Comment by tanertas** [comment]
Ok, all archs complete now. Thanks.

**Pull request closed without merging**